src/containers/item-details/index.tsx

```diff
@@ -79,12 +47,14 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 {/* <ProductCategory owner={item.owner} />
                 <ProductCollection collection={product.collection} /> */}
               </div>
-              <Button color="primary-alta" path="#">
-                Unlockable content included
-              </Button>
+              {isOwner ? (
```

Comment:
Please do not mix things up. This way we will have smaller PR and will be easier to review.